content/creator/wearables-and-emotes/manage-collections/uploading-smart-wearables.md

```diff
@@ -0,0 +1,165 @@
+
+Wearables are organized into different categories, depending on what part of an avatar they modify. Select the appropriate category for your item.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/04_category.png" width="400"/>
+
+### **Custom Thumbnails**
+
+You can add your own custom thumbnail by clicking on the camera icon and browsing your computer. **The thumbnail must be 256px square .png file with transparent background.** Collections containing thumbnails without transparent backgrounds will not be accepted by the Curation Committee.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/06_thumbnail.png" width="400"/>
+
+{{< hint warning >}}
+⚠️ Having a good render of your wearable is crucial in making it more appealing to potential users in the marketplace. **It's important to avoid adding any graphics other than the wearable itself, because this may cause the curation committee to reject it.**
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/07_thumbnail.png" width="800" />
+{{< /hint >}}
+
+### **Update Video Showcase**
+
+You can update your video showcase by clicking on the camera icon and browsing your computer. **The video max file size is 4MB and max duration is 15 seconds**.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/06_video.png" width="400"/>
+
+### **Properties**
+
+Next to the thumbnail you're going to find the properties of your wearable, number of triangles of your model, number of materials and textures.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/08_properties.png" width="400"/>
+
+## **Uploading Mouth, Eyes and Eyebrows**
+
+The mouth, eyes and eyebrows category have a different behaviour in the editor because these are just .png files. To upload these just drag and drop the png file as a transparent image (256X256 pixels). Mouth is going to be automatically tinted by skin color, same for the eyebrows tinted by the hair color.
+
+{{< hint warning >}} If you want the asset to be masked, so a part of the mouth or eyebrows is not affected by the tinting, upload a zip file with both the png and the mask files. Remeber that the mask file should have a suffix "\_mask" in order to work. {{< /hint >}}
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/02_mouth_wearable.png" width="300"/>
+
+After that uploading your wearables you will end up with a screen like this, that shows the items in your collection.
+
+<img src="/images/wearables-and-emotes/uploading-smart-wearables/11_list_wearables.png" width="900"/>
+
+## **Setting the Price of Your Smart Wearables**
+
+You can set the price of your wearable by clicking on **_Set Price_**. This can all be edited anytime, so don’t worry if you want to change it later on. Prices are set in MANA. Remember that when you mint wearables, they are minted directly on Matic/Polygon. When a user purchases your item, the transaction will be conducted in Matic/Polygon MANA.
```

Comment:
```suggestion
You can set the price of your wearable by clicking on **_Set Price_**. This can all be edited anytime, so don’t worry if you want to change it later. Prices are set in MANA. Remember that when you mint wearables, they are minted directly on Matic/Polygon. Indeed, when a user purchases your item, the transaction will be conducted in Matic/Polygon MANA.
```